docs/javascript/browser-apis-and-jquery.md

```diff
@@ -1143,3 +1143,175 @@ When you select an element from the DOM, there are many properties that live on
 ![attr example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/attr.gif?raw=true){ width="100%" }
 <figcaption></figcaption>
 </figure>
+
+
+## Changing an element's content
+
+#### `textContent` property sets or return the text content of an element
```

Comment:
I think this would read better as:

> The `textContent` property sets or returns the text...

Also, the heading should probably be:

#### `element.textContent`

Then, you'd write the description underneath.